styles/globals.css

```diff
@@ -1,18 +1,28 @@
 html,
 body {
-  padding: 0;
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
+    padding: 0;
+    margin: 0;
+    font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
     Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
 }
 
 a {
-  color: inherit;
-  text-decoration: none;
+    color: inherit;
+    text-decoration: none;
 }
 
 * {
-  box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+.username {
+    color: #000000;
+    font-family: 'Lato', sans-serif;
+    font-size: 32px;
+    font-weight: 300;
+    line-height: 58px;
```

Comment:
These can be shortened using [font](https://developer.mozilla.org/en-US/docs/Web/CSS/font) property
```css
font: 300 32px/58px 'Lato', sans-serif;
```